keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -805,12 +836,22 @@ Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
+During the beta phase, the following test will be manually performed:
```

Comment:
I wonder if the [cri proxy](https://github.com/kubernetes/kubernetes/pull/127731) work could work for this. I think having serial e2e_node tests that actually test these situations could be useful

Reply:
Yeah, cri proxy could help inject failures or mock cri behavior in e2e_node tests.

But, this section focuses upgrade -> downgrade scenario only for kubernetes components(kube-apiserver and kubelet). Although I think cri proxy could help to test upgrade/downgrade CRI runtime, this seems to be out of the scope of this section. WDYT?

Reply:
good point I think I misunderstood